src/dialogs/more-info/ha-more-info-history.ts

```diff
@@ -95,6 +95,7 @@ export class MoreInfoHistory extends LitElement {
       clearInterval(this._historyRefreshInterval);
       this._historyRefreshInterval = window.setInterval(() => {
         this._getStateHistory();
+        this._getLogBookData();
```

Comment:
This is not cached like history is it? We should only get the logbook records since the last time we fetched them.